src/app/service/game-control.service.ts

```diff
@@ -77,7 +135,67 @@ export class GameControlService {
       .subscribe();
   }
 
+  reload() {
+    this.gameActive.next(true);
+    this.updateRound();
+    this.startWithCross = !this.startWithCross;
+    this.activePlayer.next(this.togglePlayer());
+    this.gameBoard.next(new Array(9).fill(Math.random(), 0, 9));
```

Comment:
Why Math.rand?

Reply:
assigning a new Array(9) don't reload the view, so i fill it with random numbers 